doc/howto/benchmarking-time.md

```diff
@@ -0,0 +1,45 @@
+---
+jupytext:
+  notebook_metadata_filter: myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+myst:
+  html_meta:
+    description lang=en: Display cell runtime in JupyterLab
+    keywords: jupyter, jupyterlab, sql
+    property=og:locale: en_US
+---
+
+# Benchmarking run-time in JupyterLab
+To record the time taken to run each cell
+in JupyterLab, we suggest using `jupyterlab-execute-time` 
+
+# Install jupyterlab-execute-time
```

Comment:
please change this and the other headings to be H2 headings, usually, we have an H1 and the top and then H2 sub-sections, sometimes even H3

```suggestion
## Installation
```